pacemaker.spec.in

```diff
@@ -403,14 +403,26 @@ rm -rf %{buildroot}
 %postun
 %systemd_postun_with_restart pacemaker.service
 
+%pre remote
+systemctl --quiet is-active pacemaker_remote
+if [ $? -eq 0 ] ; then
+    touch %{_localstatedir}/lib/rpm-state/%{name}.restart_pacemaker_remote
```

Comment:
Ha, not my idea, copying `GConf2` behavior, state that directory as owned by `-remote` and then you can forget about that empty (since what you create there is what you normally remove as well) directory being present, it will get dropped when `-remote` is uninstalled.  Extend to any other subpackage when there's a need.